default_shardings/llama-3.yaml

```diff
@@ -0,0 +1,28 @@
+
+# Sharding config for llama-3
+# Sharding should either be an int between 0 and rank - 1
+# signifying the axis to shard or -1 / null signifying replicated
+
+
+freqs_cis : -1 #  torch.complex64 (2048, 64)
+tok_embeddings.weight : 0 #  torch.float32 (vocab_size, 4096)
```

Comment:
The sharding file seems to be the same as llama-2. What's the difference between the llama-2 and llama-3 sharding file?

From the change in `convert_checkpoints.py`, it seems that llama-3 weight is sharded in a different way. This sharding file is only used for model sharding during runtime. 

If this is the case, we don't need to have another sharding yaml file.

Reply:
The tok_embeddings.weight is sharded differently between llama-2 and llama-3. For llama-2, embeddings are sharded along axis 1 and for llama-3, they are sharded along axis 0. But I agree, that it shouldn't make a difference in accuracy during runtime. If you think that it is better to keep the same sharding for both of them then I can revert this change.

Reply:
they shouldnt be sharded differently -- the only difference would be performance; lets run with both and keep the faster one.